pgjdbc/src/main/java/org/postgresql/ds/common/BaseDataSource.java

```diff
@@ -1465,10 +1465,10 @@ public void setProperty(PGProperty property, @Nullable String value) {
       return;
     }
     switch (property) {
-      case PG_HOST:
+      case HOST:
```

Comment:
This is backward incompatible change. Old code might still call `setProperty(PGProperty.PG_HOST, ...)` and it should continue to work as expected.

Reply:
Thank you for finding!
Added fix and test case in commit: https://github.com/pgjdbc/pgjdbc/pull/2646/commits/4b294497f94545151bfd73c62407d5fbe266d7d7